slash/css/src/Form/MultiSelect/MultiSelect.scss

```
@@ -7,15 +7,52 @@
 }
 
 .af-form__select {
+  --select-border-color: transparent;
+
+  .react-select > div {
+    border-color: var(--select-border-color);
+  }
+
   &--error {
-    .react-select > div {
-      border-color: common.$color-red-axa;
-    }
+    --select-border-color: var(--red30);
   }
 
   &--warning {
-    .react-select > div {
-      border-color: common.$color-orange-dark;
+    --select-border-color: var(--orange40);
+  }
+}
+
+.react-select {
+  .multi-value-limit {
+    border-radius: 0.25rem !important;
```

on utilise pas de important normalement

---

Oui je me doute, mais ici je mets le style par défaut du composant via la propriete style et je surchage via class donc ça ne passe pas prioritaire. Il faudrait que je vire le borderRadius de l'objet style mais ça fait un truc de ce style et borderRadius sera jamais utilisé. Tu préfères quoi ?
```suggestion
    const { borderRadius, ...restStyles } = getStyles(
      "multiValueLabel",
      props as MultiValueProps<Option, boolean, GroupBase<Option>>,
    ) as CSSProperties;
```

---

De coup oui vu que malheureusement c'est du style inline et le seul moyen de faire autrement qu'un important c'est comme cela.
Peut-être mettre un commentaire pour préciser pourquoi récupérer borderRadius sans l'utiliser.